Weave Agents launch (#2647)
wandb/docs: support/weave/articles/what-is-the-difference-between-weave-op-and-weave-sdk-for-agents.mdx

---
title: "What is the difference between @weave.op and weave.start_session?"
keywords: ["Agents", "Tracing", "weave.op"]
---

[`@weave.op`](/weave/guides/tracking/tracing) traces individual Python functions and surfaces results in the **Traces** tab. 

Weave's [agentic tracing](/weave/guides/tracking/trace-agents) (`weave.start_session`, `weave.start_turn`, `weave.start_llm`, `weave.start_tool`) models a full multi-turn agent conversation and surfaces results in the **Agents** tab. 

> Both are part of the same `weave` API — the right choice depends on whether you're tracing functions and pipelines,or weather your application has agent sessions and turns.

Reviewer: ```suggestion
Both are part of the same `weave` API — the right choice depends on whether you're tracing functions and pipelines, or a multi-turn agentic application.
```
